app/assets/javascripts/distributions_and_transfers.js

```diff
@@ -94,11 +94,37 @@ $(function() {
     }
   );
 
-  $(function() {
+  $(function () {
     if (storage_location_required && !control.val()) {
       $("#__add_line_item").addClass("disabled");
     }
 
     request_storage_location_and_populate_item(default_item);
   });
+
+  $('#new_distribution').submit(function (e) {
+    e.preventDefault();
+
+    const year = $('select#distribution_issued_at_1i option:selected').val();
+    const month = $('select#distribution_issued_at_2i option:selected').val();
+    const day = $('select#distribution_issued_at_3i option:selected').val();
+
+    if (year && month && day) {
+      let submit;
+      const asked = $(this).data('asked');
+
+      const selectedDate = new Date(year, parseInt(month - 1, 0), day);
```

Comment:
I think you don't need the parseInt in this case, btw, if you gonna need it, may the `-1` should be outside of this function, shouldn't it? 

Ex:
```js
const selectedDate = new Date(year, parseInt(month, 0) - 1, day);
// or....
const selectedDate = new Date(year, month - 1, day);
```

Because on Javascript, even on Strings, the subtraction operator is available

```
> '03' - 1
< 2
```

Reply:
Really... It went unnoticed. Thanks!